pkg/config/loader.go

```diff
@@ -147,6 +149,39 @@ func LoadClient(cfg *Config) (kubernetes.Interface, error) {
 	return clientset, nil
 }
 
+func OutOfClusterClient() (kubernetes.Interface, error) {
```

Comment:
I believe you can conditionalize to detect in cluster and fallback and we can use in any location then.  

Also //Comment Exported.